text/0085-api-keys.md

```diff
@@ -457,4 +457,5 @@ The maximum size of the API key storage layer is `100GB`.
 - A restriction on the maximum offset/limit.
 - Add search parameters restrictions for an API Key.
 - Add rfc2822 format expression for `expiredAt` field. e.g. `Wed, 18 Feb 2022 23:16:09 GMT`
-- Add an alias that can only be associated to one API Key to retrieve it easily on client side. e.g. `GET /keys/:uid_or_alias`
```

Comment:
So, do we want to add a new action and don't keep the `stats` action?

Reply:
Since the `stats` endpoint is not removed, we want to keep the `stats` action. However, if we have time to add the `metrics.get` action before the v0.29, I'm all for it, it's cleaner.

Reply:
And do we add it for the admin key rights by default?

Reply:
Yes!